coreos-koji-tagger/coreos_koji_tagger.py

```diff
@@ -49,7 +49,7 @@
 ARCHES = ['x86_64', 'aarch64', 'ppc64le', 's390x']
 
 # We are processing the org.fedoraproject.prod.github.push topic
-# https://apps.fedoraproject.org/datagrepper/raw?topic=org.fedoraproject.prod.github.push&delta=100000
+# https://apps.fedoraproject.org/datagrepper/v2/search?topic=org.fedoraproject.prod.github.push&contains=fedora-coreos-config&delta=100000
```

Comment:
I'm not seeing any messages at this new URL, but maybe there just haven't been any recently?

Reply:
correct. I just enabled the push event to be sent to the new webhook yesterday.